python/paddle/fluid/framework.py

```diff
@@ -201,7 +212,7 @@ def _debug_string_(proto, throw_on_error=True):
     return proto.__str__()
 
 
-class Variable(object):
+class Variable(core.VarBase):
```

Comment:
I guess we don't have to merge the ```Variable``` in Fluid and Imperative. Maybe only ```VarDesc``` and ```OpDesc``` should be shared?